From f3a476cf1da06c99aa1dff09c4e8829a9b9db33c Mon Sep 17 00:00:00 2001
From: Kartatz <105828205+Kartatz@users.noreply.github.com>
Date: Fri, 14 Feb 2025 02:15:49 +0100
Subject: [PATCH] Fix libquadmath build on musl

---
 libquadmath/math/sqrtq.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/libquadmath/math/sqrtq.c b/libquadmath/math/sqrtq.c
index a58998a..37e6392 100644
--- a/libquadmath/math/sqrtq.c
+++ b/libquadmath/math/sqrtq.c
@@ -1,7 +1,7 @@
 #include "quadmath-imp.h"
 #include <math.h>
 #include <float.h>
-#if __has_include("../../libgcc/soft-fp/soft-fp.h") \
+#if __has_include("../../libgcc/soft-fp/this-does-not-exist.h") \
     && __has_include("../../libgcc/soft-fp/quad.h") \
     && defined(FE_TONEAREST) \
     && defined(FE_UPWARD) \
@@ -12,7 +12,7 @@
 #if defined(_ARCH_PPC) && !defined(__LONG_DOUBLE_IEEE128__)
 #define TFtype __float128
 #endif
-#include "../../libgcc/soft-fp/soft-fp.h"
+#include "../../libgcc/soft-fp/this-does-not-exist.h"
 #include "../../libgcc/soft-fp/quad.h"
 #endif
 
-- 
2.36.6

